src/main/java/ch/wisv/events/ChConnectSecurityConfiguration.java

```diff
@@ -137,7 +138,9 @@ public AuthenticationProvider oidcAuthenticationProvider() {
                 CHUserInfo info = (CHUserInfo) userInfo;
 
                 if (properties.getAdminGroups().stream().anyMatch(info.getLdapGroups()::contains)) {
-                    return ImmutableSet.of(ROLE_ADMIN, ROLE_USER);
+                    return ImmutableSet.of(ROLE_ADMIN, ROLE_USER, ROLE_COMMITTEE);
+                } else if (info.getLdapGroups().stream().filter(group -> group.equals("users")).count() > 0) {
```

Comment:
`info.getLdapGroups().contains(“users”)`?

Reply:
Ja maar ik wil juist users niet meetellen toch?

Reply:
Dan mis je de negatie in het filterpredikaat?

Reply:
Oeps, bij deze gefixt